leg_detector/src/leg_detector.cpp

```diff
@@ -37,9 +37,11 @@
 #include <leg_detector/laser_processor.h>
 #include <leg_detector/calc_leg_features.h>
 
-#include <opencv/cxcore.h>
-#include <opencv/cv.h>
-#include <opencv/ml.h>
+//#include <opencv/cxcore.hpp>
+//#include <opencv/cv.h>
+//#include <opencv/ml.h>
```

Comment:
```suggestion
```

Reply:
Np, you can! I let it just to show to others what libraries can replaced but I saw that they can see the changes from commits